src/locales/fr.json

```
@@ -473,7 +473,7 @@
     "configuration": "Configuration",
     "since": "depuis",
     "all": "Tout",
-    "app_state_low_quality": "faible qualité",
+    "app_state_low_quality": "niveau de qualité d'intégration de l'application : bas",
```

```suggestion
    "app_state_low_quality": "niveau de qualité d'intégration de l'application : bas",
```

This is supposed to be a label displayed in the catalog app cards so it can't be that long ...